src/components/PaymentOptions/index.js

```diff
@@ -83,10 +84,20 @@ const PaymentOptionsUI = (props) => {
     setCardData,
     onPlaceOrderClick,
     setCreateOrder,
-    handlePlaceOrder
+    handlePlaceOrder,
+    paymethods
   } = props
   const [, t] = useLanguage()
   const [{ token }] = useSession()
+  const [{loading}] = useOrder()
```

Comment:
For now we haven't linter, but please try to follow the code already done, use space `const [{ loading }] = useOrder()`


Also add some id to this loading in order to differentiate. ex: `loading: loadingOptions`